sdk/scala-sdk-protobuf/src/main/scala/kalix/scalasdk/impl/InternalContext.scala

```diff
@@ -16,8 +16,8 @@
 
 package kalix.scalasdk.impl
 
-import kalix.javasdk.Metadata
-import kalix.javasdk.impl.MetadataImpl
+import kalix.javasdk.impl
```

Comment:
better use the package path from/including `javasdk` to make it clear, `impl.MetadataImpl` doesn't say much, but `javasdk.impl.MetadataImpl` shows that it's from the other "side"

Reply:
totally.
done.